entries/Europa-rescue/game.json

```diff
@@ -0,0 +1,27 @@
+{
+    "developer": "godai / Gniazdo Światów",
+    "files": [
+        {
+            "default": true,
+            "filename": "Europa rescue.gbc",
```

Comment:
Filename does not correspond to the gbc file (the R is not lowercase). Check the CI: https://github.com/gbdev/database/runs/5175968279?check_suite_focus=true

Reply:
Fixed this in the manifest.